codec-http2/src/test/java/io/netty/handler/codec/http2/HpackDecoderTest.java

```diff
@@ -430,40 +454,6 @@ public void testLiteralNeverIndexedWithLargeValue() throws Http2Exception {
         decode(sb.toString());
     }
 
-    @Test
-    public void testDecodeLargerThanMaxHeaderListSizeButSmallerThanMaxHeaderListSizeUpdatesDynamicTable()
-        throws Http2Exception {
-        ByteBuf in = Unpooled.buffer(300);
-        try {
-            hpackDecoder.setMaxHeaderListSize(200, 300);
-            HpackEncoder hpackEncoder = new HpackEncoder(true);
-
-            // encode headers that are slightly larger than maxHeaderListSize
-            // but smaller than maxHeaderListSizeGoAway
-            Http2Headers toEncode = new DefaultHttp2Headers();
-            toEncode.add("test_1", "1");
-            toEncode.add("test_2", "2");
-            toEncode.add("long", String.format("%0100d", 0).replace('0', 'A'));
-            toEncode.add("test_3", "3");
-            hpackEncoder.encodeHeaders(1, in, toEncode, NEVER_SENSITIVE);
-
-            // decode the headers, we should get an exception, but
-            // the decoded headers object should contain all of the headers
-            Http2Headers decoded = new DefaultHttp2Headers();
-            try {
-                hpackDecoder.decode(1, in, decoded);
```

Comment:
is there a reason for the method movement? if not, can you undo this movement of this unit test and just add the extra parameter here?

Reply:
good catch, I'll move that back.

Reply:
- [x] Move